zktrie/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "MIT OR Apache-2.0"
 [dependencies]
 halo2_proofs = { git = "https://github.com/privacy-scaling-explorations/halo2.git", tag = "v2023_02_02" }
 mpt-circuits = { package = "halo2-mpt-circuits", git = "https://github.com/scroll-tech/mpt-circuit.git", tag = "v0.7.0" }
-zktrie = { git = "https://github.com/scroll-tech/zktrie.git", branch = "v0.6" }
+zktrie = { git = "https://github.com/scroll-tech/zktrie.git" }
```

Comment:
do we need a new branch?

Reply:
Yes, after test we would push `v0.7` for the new zktrie